packages/synapse-react-client/src/lib/containers/ModalDownload.tsx

```diff
@@ -126,59 +126,59 @@ export default class ModalDownload extends React.Component<
   }
 
   render() {
-    const closeBtn: React.CSSProperties = {
-      position: 'absolute',
-      top: 10,
-      right: 10,
-      zIndex: 10,
-    }
     const spinnerStyle: React.CSSProperties = {
       height: 50,
       width: 50,
       backgroundSize: 50,
     }
     return (
-      <Modal
-        animation={false}
-        show={true}
-        onHide={this.props.onClose}
-        backdrop="static"
-      >
-        <Modal.Body>
-          <button style={closeBtn} onClick={this.props.onClose}>
-            <IconSvg icon="close" />
-          </button>
-          <Form
-            schema={formSchemaArray[this.state.step]}
-            uiSchema={formSchemaUIArray[this.state.step]}
-            onChange={this.handleChange}
-            formData={this.state.formData}
-            onSubmit={this.handleSubmit}
-          >
-            {this.state.isLoading && (
-              <div className="SRC-centerAndJustifyContent">
-                <div className="SRC-center-text">
-                  <p> Creating the File </p>
-                  <div style={spinnerStyle} className="spinner" />
-                  <p> Loading... </p>
+      <DialogBase
+        open={true}
+        onCancel={this.props.onClose}
+        title="Download query results"
+        content={
+          <>
+            <Form
+              id="modal-download-form"
```

Comment:
Add a form id so the buttons can live in the actions section of the dialog (outside of the form), but still submit the form on click